### ISSUE TITLE
Can I read config files without sections?

### ISSUE THREAD

**Issue by ffirenze** [cemdervis/SharpConfig#85]
I've tried a simple code, reading some linux files without sections:

```
using SharpConfig;
    class Program
    {
        static void Main(string[] args)
        {
            Configuration config = Configuration.LoadFromFile("/etc/default/grub");
            foreach (var item in config)
            {
                Console.WriteLine(item.ToString());
            }
        }
    }
```

But seems like the load method requires a section formatted configuration file:

`SharpConfig.ParserException: Line 6: The setting 'GRUB_DEFAULT' has to be in a section.`

PS: I suggest move the static settings (ArrayElementSeparator , CultureInfo, IgnoreInlineComments....) to a wrapper, or a set of parameters on load method, because in a multi-threading environment static members are not safe.

Regards!

**Comment by 2art** [comment]
Seems like this is not possible right now, but a "default section" (i.e. keys before the first section in the file) could be easily implemented I guess.

I will look into it soon, unless someone beats me to it.

**Comment by cemdervis** [comment]
Fixed with version 3.2.9.
